documentation/concepts/device_scheduler.rst

```diff
@@ -0,0 +1,151 @@
+:orphan:
+
+.. _device_scheduler:
+
+Device Scheduler
+===========
+
+Introduction
+--------------
+This generic device scheduler is able to handle an EMS with multiple devices, with various types of constraints on the EMS level and on the device level,
+and with multiple market commitments on the EMS level.
+
+A typical example is a house with many devices. The commitments are assumed to be with regard to the flow of energy to the device (positive for consumption, negative for production). In practice, this generic scheduler is used in the **StorageScheduler** to schedule a storage device.
+    
+The solver minimises the costs of deviating from the commitments.
+
+
+
+Notation
+---------
+
+================================ ================================================ ==============================================================================================================  
+Symbol                              Variable in the Code                           Description
+================================ ================================================ ==============================================================================================================  
+:math:`\Delta_{up}(c,j)`              commitment_upwards_deviation                       Upwards deviation from the power commitment of the EMS.
+:math:`\Delta_{down}(c,j)`            commitment_downwards_deviation                     Downward deviation from the power commitment of the EMS.
+:math:`\Delta Stock(d,j)`                           n/a                                  Change of stock of device :math:`d` in the :math:`j` time period.
+:math:`Price_{up}(c,j)`               up_price                                           Price of incurring a upwards deviations in commitment :math:`c` in the :math:`j` time period.
```

Comment:
"in time period j" reads better. Likewise in other lines.

Reply:
Furthermore, we should distinguish verbally between the interpretation of j as a time period or as an instantaneous moment at the end (I think) of time period j. For example, j used in flow constraints should speak of *time period j*, whereas j used in a stock constraint should speak of *the end of time period j*.

Reply:
True, this needs clarification. I added a note in the description of the indices and a worked on the descriptions of the parameters/variables.